### DISCUSSION TITLE
too many cops on the colorblind

### DISCUSSION THREAD

**Discussion by WitCinema** [community/community#62944] [Accessibility]
### Select Topic Area

Bug

### Body

cop back to the error 
find the mark code '
copy back to the first instance I
only blew into Feynman 
hustle back to the hustle

**Comment by ghostinhershell** [comment]
Hey @WitCinema, thanks so much for reaching out in the Accessibility community. I'm having a hard time understanding your question or feedback here and I want to make sure that we can help. Do you mind giving me a bit more information?

**Comment by ghostinhershell** [comment]
Hey @WitCinema, since it's been about a week since this discussion has been updated, I'm going to go ahead and close this conversation. Feel free to reopen this discussion if you have any follow-up information or feedback.
